src/common/mem.c

```diff
@@ -186,9 +186,19 @@ set_slab_desc(struct umem_pool *ph_p, struct umem_slab_desc *slab)
 		davslab.units_per_block = 1000;
```

Comment:
I'm wondering if using an arbitrary fixed 'units_per_block' for all unit sizes is optimal? My concern is that if the slab internal allocation granularity is decided by this 'units_per_block', more space could be wasted by internal fragmentation when the slab with large unit size has only few allocations? Should we choose this 'units_per_block' by slab unit size?